godot-macros/src/derive_godot_class.rs

```diff
@@ -248,6 +265,18 @@ fn make_exports_impl(class_name: &Ident, fields: &Fields) -> TokenStream {
             let setter = proc_macro2::Literal::from_str(&exported_field.setter).unwrap();
             let field_type = exported_field.field.ty.clone();
 
+            let ExportHint {
+                hint_type,
+                description,
+            } = exported_field.hint.clone().unwrap_or(ExportHint {
+                hint_type: ident("PROPERTY_HINT_NONE"),
+                description: "".to_string(),
+            });
```

Comment:
We could probably add a 2nd constructor `ExportHint::none()`, then we wouldn't need to repeat the specifics in codegen.
```suggestion
            } = exported_field.hint.clone().unwrap_or(ExportHint::none());
```